### DISCUSSION TITLE
Questions about verification of new energy equipment

### DISCUSSION THREAD

**Discussion by xjlcode** [CURENT/andes#434] [Data Format]
Dear Dr.Cui,
   I have a small question regarding your renewable energy model while using Andes recently. Specifically, I'm wondering if I can validate your model's correctness in PSS/E regarding kundur_wtdta1. This is because I haven't seen any .raw or .dyr files that contain new energy equipment in your model.
  My question is that I changed one generator in the Kundur system to a wind turbine in PSS/E version 35.3, just like your Excel spreadsheet, but I cannot get the same results as Andes. So Could you please give me some advice on it?
  Thank you!

**Comment by cuihantao** [comment]
I have not created PSS/E files to verify the kundur_wtdta1 case. I would appreciate it and provide guidance if you would like to contribute.

It is the same `.raw` file, and you can create the dyr file using the parameters as in the Excel spreadsheet. You can apply a line trip event and check the output.

**Reply by xjlcode** [reply]
Yes, it seems like you did not verify renewable energy model in PSS/E because i have not seen the .dyr files. I am trying to verify it by debugging model parameters of andes and PSS/E.

**Comment by cuihantao** [comment]
It would be very helpful if you could take on it.

You might start from the same power flow solution and use converter models, namely, REGCA1, and REECA1. REECA2 has many control flags, and starting with the ones with basic functionality can simplify the process. After matching the results, you can then add the models for wind generators.

Most of the time, the mismatches are due to parameters and limiters. Enabling the minimum necessary control flags can help to reduce the number of affecting parameters. For limiters, it’s usually okay to ignore them first by setting a very large range.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Reply by xjlcode** [reply]
Thanks for your reply.

I have put your converter files such as REGCA1, REECA1 into PSS/E35.3.  In other words, I just changed one of syn generators to wind model in kundur_full.dyr. However, I cannot get the same results as andes in PSS/E.

There may be some errors in my parameters. I wonder if I change the swing bus or alter the capacity of the generator can fix it.

 Has andes taken some idealistic measures? Please let me know.

**Reply by jinningwang** [reply]
Comments:
1. Check if the power flow results are consistent.
2. Check if there is any parameter autocorrection after TDS initialization (`ss.TDS.init()`). If so, a possible reason can be that PSSE and ANDES have different parameter corrections.
3. Check if the contingency you used is consistent.
4. Check if the simulation settings are the same, such as step size.
5. I don't think it comes first that alters the swing bus or generator capacity.

At last, it is not guaranteed that PSSE and ANDES will have the exact same results even with the exact same input.

**Comment by xjlcode** [comment]
I seem to have found the answer to the problem: the initial voltage and phase of the raw file and xlsx file in the kundur folder do not match, resulting in the initial non-convergence problem. And I checked that the initial voltage and phase of the two files in the ieee14 folder are consistent.
I think the data in the raw file is the correct initial data after calculating the pflow.

**Reply by jinningwang** [reply]
Thanks for your update.
A tip is that, ANDES takes PSS/E input, and you can easily convert the PSS/E case to ANDES xlsx format for further use in ANDES. This might save your time in the future.

Regards,
Jinning

**Reply by xjlcode** [reply]
Yes, I know this way.
Can ANDES convert the xlsx format to the PSS/E case?

**Reply by jinningwang** [reply]
ANDES doesn't have such a function now.
